x/accounts/internal/prefixstore/prefixstore.go

```diff
@@ -0,0 +1,223 @@
+// Package prefixstore provides a store that prefixes all keys with a given
+// prefix. It is used to isolate storage reads and writes for an account.
+// Implementation taken from cosmossdk.io/store/prefix, and adapted to
+// the cosmossdk.io/core/store.KVStore interface.
+package prefixstore
```

Comment:
unfortunately we don't have a prefix store implementation which complies with core.store.KVStore ....

Reply:
for now this is probably fine with the ongoing work of storev2, in the future we can migrate to the store one